### PR TITLE
Add make to to build Sphinx docs and configuration fix for changelogs generation

### PR DESCRIPTION
**Pull request by tonyastrofig**
This addresses issue #12. It adds the apt-get update and install make to the Dockerfile

### CODE REVIEW

**Comment by tonyastrofig** [comment]
@ehsteve, I created an issue:
https://github.com/HERMES-SOC/hermes_instrument/issues/12
and then created a branch named "add_make".  I only modified 1 file, the dockerfile.  I checked that the container built and was able to make the Sphinx documentation.  All checks have passed.  So I think it is ready for your review.

File: .devcontainer/Dockerfile [hunk 1]
```diff
@@ -8,6 +8,8 @@ FROM mcr.microsoft.com/vscode/devcontainers/base:ubuntu-20.04
 RUN apt-get update && \
     apt-get -y upgrade && \
     apt-get install --no-install-recommends -y python3.8 python3-pip python3.8-dev pylint
+RUN apt-get update && \
```
Comment (ehsteve) [inline .devcontainer/Dockerfile:11]: why run update again? i Would just add it the list of commands in the run command above

**Review by ehsteve — CHANGES_REQUESTED** [review]
This does add the make command but it adds an unnecessary update command.

**Review by ehsteve — APPROVED** [review]

File: .devcontainer/Dockerfile [hunk 2]
```diff
@@ -7,6 +7,7 @@ FROM mcr.microsoft.com/vscode/devcontainers/base:ubuntu-20.04
 # RUN bash -i -c 'pip3 install --user --no-use-pep517 -e .[all]'
 RUN apt-get update && \
     apt-get -y upgrade && \
+    apt-get install make && \
```
Comment (tonyastrofig) [inline .devcontainer/Dockerfile:10]: Correctly installs make for sphinx documentation

File: docs/Makefile [hunk 3]
```diff
@@ -14,6 +14,10 @@ help:
 
 .PHONY: help Makefile
 
+clean:
```
Comment (tonyastrofig) [inline docs/Makefile:17]: make clean successfully removes __build directory

**Comment by tonyastrofig** [comment]
@ehsteve, your changes all look good, but since I generated the pull request I can't approve it.  Once you re-approve it, do I hit the "Merge pull request" button?

**Review by ehsteve — APPROVED** [review]

**Pull request merged**